common/djangoapps/third_party_auth/tests/specs/base.py

```diff
@@ -761,14 +764,16 @@ def test_full_pipeline_succeeds_for_signing_in_to_existing_active_account(self):
         self.set_logged_in_cookies(request)
 
         self.assert_redirect_to_dashboard_looks_correct(
-            actions.do_complete(request.backend, social_views._do_login, user=user))
-        self.assert_account_settings_context_looks_correct(account_settings_context(request), user)
+            actions.do_complete(request.backend, social_views._do_login, user=user)
+        )
+        self.assert_account_settings_context_looks_correct(account_settings_context(request))
 
     def test_signin_fails_if_account_not_active(self):
         _, strategy = self.get_request_and_strategy(
             auth_entry=pipeline.AUTH_ENTRY_LOGIN, redirect_uri='social:complete')
         strategy.request.backend.auth_complete = mock.MagicMock(return_value=self.fake_auth_complete(strategy))
-        user = self.create_user_models_for_existing_account(strategy, 'user@example.com', 'password', self.get_username())
+        user = self.create_user_models_for_existing_account(strategy,
```

Comment:
If we needed to change this, we'd want to use the hanging indent style as above. But since we don't need to change it, we'll leave the awful existing style. 😉